telegram/constants.py

```diff
@@ -28,6 +28,7 @@
 
     * Most of the constants in this module are grouped into enums.
 """
+# pylint: disable=invalid-enum-extension
```

Comment:
@harshil21 can we add a reference to the a gh thread which explains why this is here and when we might be able to remove it?